Mathlib/Analysis/BoxIntegral/Basic.lean

```diff
@@ -690,7 +690,8 @@ theorem integrable_of_bounded_and_ae_continuousWithinAt [CompleteSpace E] {I : B
       simpa only [edist_le_ofReal (le_of_lt ε₁0), dist_eq_norm, hJ.1] using ineq.trans (hr x hx)
     refine (norm_sum_le _ _).trans <| (sum_le_sum this).trans ?_
     rw [← sum_mul]
-    trans μ.toBoxAdditive I * ε₁; swap; linarith
+    trans μ.toBoxAdditive I * ε₁; swap
```

Comment:
This "swap and handle side goal idiom" could also be replaced by "pick_goal 2 => linarith" or similar. Just tell me your favourite bikeshed colour :-)